### PR TITLE
feat: bump kubelet config to ga + minor doc fixes

### PR DESCRIPTION
**Pull request by DrFaust92**

### CODE REVIEW

**Comment by DrFaust92** [comment]
[apeabody](https://github.com/apeabody) need another comment here after the merge 😸

File: cluster.tf [hunk 1]
```diff
@@ -702,6 +702,20 @@ resource "google_container_node_pool" "pools" {
...
+      )) != 0 ? [1] : []
+
+      content {
+        cpu_manager_policy   = lookup(each.value, "cpu_manager_policy", "static")
+        cpu_cfs_quota        = lookup(each.value, "cpu_cfs_quota", null)
+        cpu_cfs_quota_period = lookup(each.value, "cpu_cfs_quota_period", null)
+        pod_pids_limit       = lookup(each.value, "pod_pids_limit", null)
+      }
+    }
+
```
Comment (apeabody) [inline cluster.tf:718]: nit: we can remove this extra line

```suggestion
```

File: cluster.tf [hunk 2]
```diff
@@ -961,6 +975,20 @@ resource "google_container_node_pool" "windows_pools" {
...
```
Comment (apeabody) [inline cluster.tf:991]: nit: we can remove this line, and probably even another extra line

```suggestion
```

**Review by apeabody — APPROVED** [review]
Thanks for the contribution @DrFaust92!

**Pull request merged**